### DISCUSSION TITLE
Slides from session 2: Connecting a RAG Chat App to Azure Cosmos DB

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/AI-Chat-App-Hack#33] [Announcements]
Slides: https://speakerdeck.com/kmodi/connecting-a-rag-chat-app-to-azure-cosmos-db
Recording: https://www.youtube.com/watch?v=cpbzQ-PfC4Y

**Comment by jeremykpark** [comment]
As suggested I applied to the cosmosdb grant but was rejected. I applied again and waiting to hear back but not sure how long I can keep this server going.

The question on the form asking 'What is your Azure Cosmos DB account name?' is kind of ambiguous as to what name exactly they are looking for, as there are a few account names to set this up.

**Reply by khelanmodi** [reply]
Hey Jeremy, The account name is a unique identifier for your Cosmos DB account. Kindly take a look at the screenshot here:  https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/quickstart-portal#create-account to locate your account name.

If you're using the MongoDB API, please enter the cluster name instead of the account name. I hope this helps!

**Reply by jeremykpark** [reply]
That's the account I put but I was rejected, and haven't heard back since second attempt. I explained I was taking part in the hackathon in my application. Will probably cancel Cosmos today I'll just use the other data approach :/
